wheretofindme/views.py

```diff
@@ -43,3 +45,12 @@ class IdentityViewset(viewsets.ModelViewSet):
 
     def get_queryset(self):
         return InternetIdentity.objects.filter(user=self.request.user)
+
+    @action(detail=False, methods=['POST'], name='Reorder Identities')
+    def reorder(self, request):
+        self.get_queryset().all().update(seq=None)
+        for seq, id in enumerate(request.data):
+            identity = self.get_queryset().get(id=id)
+            identity.seq = seq
```

Comment:
Django can be set to wrap every request/response cycle in a transaction, which should suffice I guess. Still, as you say, it's a lot of DB traffic, so that could be, separately, something we could fix.

Reply:
Fixed in 2ac2851, I believe.